ansible/roles_ocp_workloads/ocp4_workload_integreatly/defaults/main.yml

```diff
@@ -5,15 +5,39 @@ silent: False
 
 ocp4_workload_integreatly_documentation_link: https://docs.google.com/document/d/16mFcIjyykwY9oR0K7xRPZ0nAWm6AnhGvEBYISDlakTQ
 
+# Machine Config
+# We need to override the machine pool config to allow for a larger
+# maximum node quota. This is because on clusters with 50 users, 50
+# Fuse Online instances are created, which contain many pods.
+# On these large clusters, resources are not the bottleneck. The default quotas
+# are, as they stop most resources from ever being used.
+ocp4_workload_integreatly_machineconfigpool_name: worker
+ocp4_workload_integreatly_machineconfigpool_label_value: "true"
+
+ocp4_workload_integreatly_kubeletconfig_name: rhmi-max-pod-override
+ocp4_workload_integreatly_kubeletconfig_template: kubeletconfig-pods.yml.j2
+ocp4_workload_integreatly_kubeletconfig_pods_max: 500
+ocp4_workload_integreatly_kubeletconfig_pods_core: 500
+
 # Operator
 ocp4_workload_integreatly_user_count: 50
 ocp4_workload_integreatly_tmp_dir: /tmp
 ocp4_workload_integreatly_prefix: redhat-rhmi-
 ocp4_workload_integreatly_namespace: "{{ ocp4_workload_integreatly_prefix }}operator"
+# The actual release version. This should be treated as the only version input
+# an end user should need to override in normal circumstances.
+# This should be updated on each new release. It should always be the latest release.
 ocp4_workload_integreatly_release: "2.2.0"
-# RC's are used until full release is available
-# ocp4_workload_integreatly_git_ref: "v{{ ocp4_workload_integreatly_release }}"
-ocp4_workload_integreatly_git_ref: v2.2.0-rc1
+# A map for ocp4_workload_integreatly_release to an actual git ref
+# This allows the version itself and the git ref to differ, which is common
+# as the git ref is often prefixed with a 'v' and during release testing will
+# also contain a '-rc{N}' type suffix.
+# This mapping should be added to for each new release.
+ocp4_workload_integreatly_git_ref_map:
```

Comment:
this is just a workaround for release candidates and it gets rid of the assumption that the git ref will always be a release version prefixed with `v`

Reply:
We probably shouldn't have any release tags specified in this workload in general as it will be read in from a variable linked to the version dropdown in the workshop submission form. For development purposes we'll just need to ensure that we're specifying a release version using `-e ocp4_workload_integreatly_release=v2.2.0`

Reply:
We'll probably just map `ocp4_workload_integreatly_release` to the version dropdown value

Reply:
@pmccarthy The reason I'd added this is because the git ref (tag) and the operator release version differ:

- For the git ref `v2.2.0`, `2.2.0`, is the operator release version 
- For the git ref `v2.2.0-rc3`, `2.2.0` is the operator release version
- For the git ref `master`, `2.2.0` is the operator release version

As we really only care about proper releases, I'm happy to let `ocp4_workload_integreatly_release` be the actual git ref and then filter out the `v` for determining the actual operator release version by default though.

So, user selects `v2.0.0` and:
- Role assumes `v2.0.0` is the git ref
- Role assumes `2.0.0` (filter out the `v`) is the operator release version

This would break for RC's, but as you've said, we can just override it externally locally.